### PR TITLE
Automatic update of .github/CODEOWNERS

### PR DESCRIPTION
**Pull request by LPardue**

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -1,16 +1,13 @@
 # Automatically generated CODEOWNERS
 # Regenerate with `make update-codeowners`
-draft-ietf-httpbis-bcp56bis.md mnot@mnot.net
-draft-ietf-httpbis-cache-header.md mnot@mnot.net
+ mt@lowentropy.net caw@heapingbits.net
```
Comment (LPardue) [inline .github/CODEOWNERS:3]: this doesn't seem correct because there is no draft. Not sure MT or Chris will welcome becoming codeowner for everything

Comment (chris-wood) [reply .github/CODEOWNERS:3]: 😱

Comment (LPardue) [reply .github/CODEOWNERS:3]: FWIW this commit was made via `http-extensions$ make update-codeowners`. @martinthomson or @mnot might want to figure out what caused the script to bug out

Comment (martinthomson) [reply .github/CODEOWNERS:3]: At a meeting, but I'll take a look at it when I am able.

**Comment by LPardue** [comment]
Overtaken by events thst fixed the bug

**Pull request closed without merging**